uint64 timestamp	# time at which the latest parameter was updated
bool saved		# wether the change has already been saved to disk
